contracts/FNDCart.sol

```diff
@@ -33,9 +34,14 @@ contract FNDCart is Ownable {
    * @notice Initialize the contract.
    * @param _market The Foundation market contract address on this network.
    */
-  constructor(address payable _market, address payable _feth) {
+  constructor(
+    address payable _market,
+    address payable _feth,
+    address payable _referrerTreasury
```

Comment:
nit: a @param comment (and/or one on the variable above) could be helpful to explain what this value is used for.